lib/adapters/http.js

```diff
@@ -13,33 +14,54 @@ var zlib = require('zlib');
 var VERSION = require('./../env/data').version;
 var createError = require('../core/createError');
 var enhanceError = require('../core/enhanceError');
-var defaults = require('../defaults');
 var Cancel = require('../cancel/Cancel');
+var defaults = require('../defaults');
```

Comment:
Very minor change, I could revert it if you wish: this fixes a circular dependency issue that doesn't show up here, but which reared its head when I tried adding `var getProxyForUrl = require('proxy-from-env').getProxyForUrl;` after it. In the end, I added it above following alphabetical order... but please feel free to ask me to make changes to the `require` ordering.

Reply:
Looks like it's taken care of in https://github.com/axios/axios/pull/4220 in a much better way. I'll revert moving this require to avoid unnecessary merge conflicts .